.env

```diff
@@ -0,0 +1,3 @@
+REGISTRY_NAME=''
```

Comment:
why is this file needed? seems to fail the tests

Reply:
It's default env cars used in compose. They are used when pushing images so locally should default to empty string.
Adding this allowed removing exports in the script, and not printing warning when running compose.
It's should not fail the build, I think it fails from the sporadic issue